sormas-e2e-tests/src/main/java/org/sormas/e2etests/common/StringUtils.java

```diff
@@ -0,0 +1,18 @@
+package org.sormas.e2etests.common;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringUtils {
+
+  public static String getMatchedGroupByIndexFromAString(
+      String string, String regex, int groupIndex) {
+    String returnedMatch = null;
```

Comment:
this is not a good practice to declare null objects.

Reply:
reworked. thanks.